test/test_examples_3d.jl

```diff
@@ -111,42 +124,119 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
+      linf = [0.9307979866990295, 1.4326649193456429, 1.4326649193456429, 1.4326649193456429, 12.805850412386896],
+      initial_condition=Trixi.initial_condition_sedov_self_gravity, cfl=0.25, shock_alpha_max=1.0, t_end=0.1)
+  end
+
+  @testset "taal-confirmed elixir_euler_shockcapturing_amr.jl" begin
+    # OBS! This setup does not make much practical sense. It is only added to exercise the
+    # `sedov_self_gravity` AMR indicator, which in its original configuration is too expensive for
+    # CI testing
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_shockcapturing_amr.jl"),
+      l2   = [0.022890627324485553, 0.013353127563710173, 0.013353127563710156, 0.013352146925243637, 0.08348119429398775],
+      linf = [0.3769085161148348, 0.380358363190641, 0.3803583631906434, 0.380366775575835, 1.3162027903728162],
+      maxiters=10)
+  end
+
   @testset "taal-confirmed elixir_euler_density_pulse.jl with source_terms_harmonic" begin
-  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_density_pulse.jl"),
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_density_pulse.jl"),
       l2   = [0.05719652660597408, 0.0571965266059741, 0.05719652660597407, 0.05719652660597409, 0.08579478990896279],
       linf = [0.27375961853433606, 0.27375961853433517, 0.27375961853433384, 0.2737596185343343, 0.4106394278015033],
       source_terms=Trixi.source_terms_harmonic)
   end
 
   @testset "taal-confirmed elixir_euler_ec.jl" begin
-  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_ec.jl"),
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_ec.jl"),
       l2   = [0.025101741317688664, 0.01655620530022176, 0.016556205300221737, 0.016549388264402515, 0.09075092792976944],
       linf = [0.43498932208478724, 0.2821813924028202, 0.28218139240282025, 0.2838043627560838, 1.5002293438086647])
   end
 
-  @testset "taal-confirmed elixir_mhd_alfven_wave.jl" begin
-  test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave.jl"),
-      l2   = [0.0038729054515012624, 0.00903693761037057, 0.0041729297273898815, 0.01160504558506348, 0.006241548790045999, 0.009227641613254402, 0.0034580608435846143, 0.011684993365513006, 0.0022068452165023645],
-      linf = [0.012628629484152443, 0.03265276295369954, 0.012907838374176334, 0.044746702024108326, 0.02796611265824822, 0.03453054781110626, 0.010261557301859958, 0.044762592434299864, 0.010012319622784436])
+  @testset "taal-confirmed elixir_euler_ec.jl with initial_condition=Trixi.initial_condition_constant" begin
+    test_trixi_run(joinpath(EXAMPLES_DIR, "elixir_euler_ec.jl"),
+      l2   = [5.717218008425079e-16, 6.088971423170968e-16, 6.23130776282275e-16, 7.29884557381127e-16, 5.167198077601542e-15],
+      linf = [3.885780586188048e-15, 4.454769886308441e-15, 3.219646771412954e-15, 4.884981308350689e-15, 4.440892098500626e-14],
+      initial_condition=Trixi.initial_condition_constant)
```

Comment:
These source terms should be exported (they are on `dev`).

Reply:
Fixed in a2bedcb.